source/_integrations/kaleidescape.markdown

```diff
@@ -0,0 +1,136 @@
+
+#### media_player.turn_on
+
+You can turn on a player with the `media_player.turn_on` service. Example service payload:
+
+```yaml
+- service: media_player.turn_on
+  data:
+    entity_id: media_player.kaleidescape_theater
+```
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | no      |  `entity_id` of the player
+
+#### media_player.turn_off
+
+You can turn off a player with the `media_player.turn_off` service. Example service payload:
+
+```yaml
+- service: media_player.turn_off
+  data:
+    entity_id: media_player.kaleidescape_theater
+```
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | no       |  `entity_id` of the player
+
+#### media_player.media_play
+
+You can play the currently selected media with the `media_player.media_play` service. Example service payload:
+
+```yaml
+- service: media_player.media_play
+  data:
+    entity_id: media_player.kaleidescape_theater
+```
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | no       |  `entity_id` of the player
+
+#### media_player.media_pause
+
+You can pause the currently playing media with the `media_player.media_pause` service. Example service payload:
+
+```yaml
+- service: media_player.media_pause
+  data:
+    entity_id: media_player.kaleidescape_theater
+```
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | no       |  `entity_id` of the player
+
+#### media_player.media_stop
+
+You can stop the currently playing media with the `media_player.media_stop` service. Example service payload:
+
+```yaml
+- service: media_player.media_stop
+  data:
+    entity_id: media_player.kaleidescape_theater
+```
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | no       |  `entity_id` of the player
+
+#### media_player.media_next_track
+
+You can skip to the next track with `media_player.media_next_track` service. Example service payload:
+
+```yaml
+- service: media_player.media_next_track
+  data:
+    entity_id: media_player.kaleidescape_theater
+```
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | no       |  `entity_id` of the player
+
+#### media_player.media_previous_track
+
+You can skip to the previous track with `media_player.media_previous_track` service. Example service payload:
+
+```yaml
+- service: media_player.media_previous_track
+  data:
+    entity_id: media_player.kaleidescape_theater
+```
+
+| Service data attribute | Optional | Description |
+| ---------------------- | -------- | ----------- |
+| `entity_id`            | no       |  `entity_id` of the player
```

Comment:
```suggestion
```
Only custom services for the integration need to be documented

These are already documented as part of the base media_player entity here
https://www.home-assistant.io/integrations/media_player#media-control-services
and there is no process to update them in other places when they change so we don't
want them here.

Reply:
Removed.